incubator/hnc/internal/reconcilers/object_test.go

```diff
@@ -45,6 +45,18 @@ var _ = Describe("Secret", func() {
 		cleanupObjects(ctx)
 	})
 
+	PIt("should propagate objects only to selected namespace", func() {
+		setParent(ctx, barName, fooName)
+		setParent(ctx, bazName, fooName)
+
+		// Create a secret that does NOT propogate to bazName
+		a := map[string]string{"propagate.hnc.x-k8s.io/treeSelect": "!" + bazName}
+		makeObjectWithAnnotation(ctx, "Secret", fooName, "fooScret", a)
+
+		Eventually(hasObject(ctx, "Secret", barName, "fooSecret")).Should(BeTrue())
+		Eventually(hasObject(ctx, "Secret", bazName, "fooSecret")).Should(BeFalse())
```

Comment:
This should be `Consistently`: http://onsi.github.io/gomega/#consistently

Reply:
Done